### PR TITLE
Fix typos and improve readability in documentation

### PR DESCRIPTION
**Pull request by xkons**
I carefully read all the documentation and code and I really like what you've built!
I fixed some typos and unintuitive sentence structures in the documentation along the way.

### CODE REVIEW

File: NotModified/Annotation/ReplaceWithNotModifiedResponse.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 use Webfactory\HttpCacheBundle\NotModified\VoterInterface;
 
 /**
- * This Annotation determines the last modified date over all of it's parameterising voters. This date is used by the
+ * This Annotation determines the last modified date over all of its parameterised voters. This date is used by the
```
Comment (MalteWunsch) [inline NotModified/Annotation/ReplaceWithNotModifiedResponse.php]: Lets compromise on "its parameterising", as the voters parameterise the annotation? (The voters by themselves may be parameterised, too, but that's not the point here.)

Comment (xkons) [reply NotModified/Annotation/ReplaceWithNotModifiedResponse.php]: This is a tough one for me, as I'm not sure what parameterising means. The parametirising magic you're referring to happens in ReplaceWithNotModifiedResponse::determineLastModified() right? As I see it, the Annotation _uses_ the voters, that were given to it through parameters, to find the latest "last modified" date.
Please help me understand 😇

Comment (MalteWunsch) [reply NotModified/Annotation/ReplaceWithNotModifiedResponse.php]: Oh, there's no magic in that, just a (confusingly?) shortened sentence.

The annotation has to have a parameter containing a voters description.

The annotation uses this voters description (which is parameterising the annotation) to determine the last modified date over all Voters. => The annotation uses the voters (which are parameterising the annotation) to determine the last modified date over all Voters. => The annotation uses its  parameterising voters to determine the last modified date over all Voters. => The annotation determines the last modified date over all its parameterising voters.

And you're right: the way the annotation uses the voters description is implemented in ReplaceWithNotModifiedResponse::determineLastModified(). First, constructing the Voters from the description, then querying the Voters for their individual "last modified" date, remembering the latest one.

Comment (xkons) [reply NotModified/Annotation/ReplaceWithNotModifiedResponse.php]: @MalteWunsch I understand what's happening, but I'm not sure about the linguistics. 
Said "parameterising of the annotation" happens here "$this->voters[] = $voter;", right?

There you are taking the parameters of the annotation, which you turned into a Voter object and put these objects as "parameters" into ReplaceWithNotModifiedResponse::voters

Comment (MalteWunsch) [reply NotModified/Annotation/ReplaceWithNotModifiedResponse.php]: Ah. In my readme prose, I didn't want to dive too deep into the technical details, but rather give a quick understanding of the concept and usage. Hence, I only had the annotation in its docblock form in mind:

    /**
     * @ReplaceWithNotModifiedResponse(voters = {"@app_caching_articlevoter"})
     */

and I called the ```voters = {"@app_caching_articlevoter"}``` part "parameter". I did not want to mention the ReplaceWithNotModifiedResponse class and the conversion from voter description to Voter objects at all.

**Pull request merged**